.setup/SUBMITTY_TEST.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+
+current_dir=$(pwd)
+
+cd /usr/local/submitty/GIT_CHECKOUT/Submitty/site || {
+    echo "Failed to change to the Submitty/site directory.
+    Please check if you have /usr/local/submitty/GIT_CHECKOUT/Submitty/site and valid permission"
+    exit 1
+}
+
+
+submitty_test() {
```

Comment:
Why are we creating a function and then calling it immediately afterwards?

Reply:
I'm still a bit confused about this.  Can't you just move all of this logic outside the function?

Reply:
Yea, I was just trying to make it neat, because initially I got confused by checking extra flags/arguments with "$@"
If you don't like it I can make the change.

Reply:
Can't you still do that without using the function?

Reply:
Yup, made the changes :)